packages/wonder-blocks-data/src/components/__tests__/data.test.js

```diff
@@ -132,8 +143,8 @@ describe("Data", () => {
                 );
 
                 // Assert
-                expect(fakeChildrenFn).toHaveBeenCalledTimes(2);
-                expect(fakeChildrenFn).toHaveBeenLastCalledWith({
+                // expect(fakeChildrenFn).toHaveBeenCalledTimes(2);
```

Comment:
Old test?

Reply:
Ah yes, I was having to tinker with the test cases to work out what was going on. Good catch. Thanks.